packages/core/src/App/Containers/app-notification-messages.jsx

```diff
@@ -159,7 +172,9 @@ const AppNotificationMessages = ({
 
     const notifications_sublist =
         window.location.pathname === routes.cashier_deposit
-            ? filtered_excluded_notifications.filter(message => message.key.includes('switched_to_real'))
+            ? filtered_excluded_notifications.filter(
+                  message => message.key.includes('switched_to_real') || maintenance_notifications.includes(message.key)
```

Comment:
Could this be written as
`message => [... maintenance_notifications, 'switched_to_real'].includes(message.key)`
if thats what is logically intended here.